superset-frontend/packages/superset-ui-core/src/number-format/NumberFormats.ts

```diff
@@ -35,20 +35,20 @@ const FLOAT_SIGNED = FLOAT_SIGNED_2_POINT;
 const INTEGER = ',d';
 const INTEGER_SIGNED = '+,d';
 
+const PERCENT = ',.0%';
```

Comment:
Changes the defaults. The only file using the modified constants was the Heatmap chart which was updated.